src/components/button-group/button-group.scss

```diff
@@ -113,7 +113,7 @@
     }
 
     &.mdc-chip--selected {
-        background-color: var(--mdc-theme-surface);
```

Comment:
some icon missing it now:
<img width="324" height="73" alt="Screenshot 2025-08-04 at 14 25 28" src="https://github.com/user-attachments/assets/13ea7900-a818-46de-8c67-3a0d81f54b90" />